### ISSUE TITLE
can't seem to send message to a groupchat. 

### ISSUE THREAD

**Issue by jacksnodgrass** [Norgul/xmpp-php#8]
I can do: 
`$client->message->send($message, $recipient,$msg_type);`
where $msg_type = 'chat' and send the specified message to the specified recipient. 

changing $msg_type to 'groupchat' does not get the message sent to a chat group. 

You have to FIRST do something like: 
`$client->presence->subscribe("chatroom@conference.chat.server/resource_name");`

BUT xmpp-php sends: 

```
<presence from='jacks@spark.maestros.com' to='callcenter@conference.spark.maestros.com' 
 type='subscribe' />
```

and that does not work for my openchat chat server.  I don't get an error but when I do the send() I get a 406 error. 

`<error code="406" type="modify"><not-acceptable xmlns="urn:ietf:params:xml:ns:xmpp-stanzas"/></error>`

If we send: 
`<presence from='jacks@spark.maestros.com' to='callcenter@conference.spark.maestros.com' /> `
instead... then the send() to the chat room works. 

- jack

**Comment by Norgul** [comment]
Thanks Jack for these comments and improvements. Do you have a server test account somewhere readily available so I can test it out as well?

**Comment by jacksnodgrass** [comment]
I'll see if I can get something setup on Monday that you can access. Do you use docker at all?

**Comment by Norgul** [comment]
Thanks.

Regarding Docker yes, but not here because this is meant to be the core library, and if I would get Docker container it would make much more sense to make a container with Ratchet which runs a WS server which calls the library.

**Comment by jacksnodgrass** [comment]
I was asking about docker because I have a pretty good docker-compose setup for openfire ( a xmpp based chat server ) could be used for localized testing.  - jack

**Comment by Norgul** [comment]
That would be great!

**Comment by Norgul** [comment]
Hello @jacksnodgrass,
is there anything new in setting me up the access for the server or maybe merging a openfire Docker env?

Thanks

**Comment by jacksnodgrass** [comment]
no.. sorry.. busy with work. will try and get something setup soon.
